content/article/the-rock-pi-s-or-how-to-get-gpio-input-without-a-library.md

```diff
@@ -0,0 +1,212 @@
+
+  {
+    "title"       : "The Rock Pi S; or, How to Get GPIO Input Without a Library",
+    "authors"     : ["timm-murray"],
+    "date"        : "2020-01-20T17:50:11",
+    "tags"        : [ "hardware", "linux", "gpio", "pi", "rockpi" ],
+    "draft"       : true,
+    "image"       : "/images/the-rock-pi-s-or-how-to-get-gpio-input-without-a-library/rockpi.jpg",
+    "thumbnail"   : "",
+    "description" : "There are lots of single board computers out there. GPIO (General Purpose Input Output) is what we usually use to see if something is on or off, but not all of these newer boards have a simple library for accessing them in Perl. Never fret, because most of them run Linux, and it's easy to access hardware pins with just Linux.",
```

Comment:
The site assumes descriptions are just a single sentence, apologies that wasn't obvious. I should add something about this to the contributor guide. Here's what happens with a long description:

![image](https://user-images.githubusercontent.com/1469333/72951746-8886a600-3d5d-11ea-910b-9976508227f3.png)

How about breaking the title up into:

title: The Rock Pi S Review
description: How to get GPIO without a library
?

Reply:
Those changes look good, go ahead with them.